classes/teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, name, parent_pamission, specialization)
+    super(age, name, parent_pamission: parent_pamission)
```

Comment:
- [ ] As mentioned previously kindly make sure to add a default value for the `name` and `parent_permission` parameters for the Teacher class as well.